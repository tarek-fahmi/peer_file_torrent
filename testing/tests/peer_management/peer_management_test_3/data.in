Peer Invalid Connection Attempt
./testing/bin/btide ./testing/resources/configs/config.cfg
CONNECT 127.0.0.1:0000
PEERS
DISCONNECT 127.0.0.1:0000
QUIT