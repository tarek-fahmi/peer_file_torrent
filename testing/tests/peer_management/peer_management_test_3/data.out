Unable to connect to request peer
Not connected to any peers
Unknown peer, not connected
